robots/commenter/main.go

```diff
@@ -70,10 +68,10 @@ func flagOptions() options {
 	flag.StringVar(&o.comment, "comment", "", "Append the following comment to matching issues")
 	flag.BoolVar(&o.useTemplate, "template", false, templateHelp)
 	flag.IntVar(&o.ceiling, "ceiling", 3, "Maximum number of issues to modify, 0 for infinite")
-	flag.Var(&o.endpoint, "endpoint", "GitHub's API endpoint")
-	flag.StringVar(&o.graphqlEndpoint, "graphql-endpoint", github.DefaultGraphQLEndpoint, "GitHub's GraphQL API Endpoint")
-	flag.StringVar(&o.token, "token", "", "Path to github token")
```

Comment:
Could we keep these temporarily and backfill `o.github.*` items from them to avoid breaking existing users (outside of k8s)? 

I'd prefer to do that and emit a warning about eventually (a month? two?) removing these options, to give folks at least a chance to notice and fixup their configs. With a reminder comment someone will eventually notice and remove the code after the date.

Reply:
(as written this will also break kubernetes, unless quickly followed with a successful image upgrade ...)

Reply:
I've started by wiring the old flags, but some of the fields were private. Let me give it a second try with some extra variables and syncing it back.